### PR TITLE
DR4Hub known issues update 2.7 stolostron/backlog#26797

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26797#issuecomment-1310761219

AsciiDoc is broken at line 788, if anyone wants to try to figure it out.

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -756,23 +754,16 @@ If you cannot verify, attempt to import the cluster again and verify again.
 
 Learn about {ocp} SCC at https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html#security-context-constraints-about_configuring-internal-oauth[Managing Security Context Constraints (SCC)], which is an additional configuration required on the managed cluster.
 
-Different deployments have different security context and different service accounts.
-The subscription operator cannot create an SCC automatically.
-Administrators control permissions for pods.
-A Security Context Constraints (SCC) CR is required to enable appropriate permissions for the relative service accounts to create pods in the non-default namespace:
+Different deployments have different security context and different service accounts. The subscription operator cannot create an SCC automatically. Administrators control permissions for pods. A Security Context Constraints (SCC) CR is required to enable appropriate permissions for the relative service accounts to create pods in the non-default namespace. To manually create an SCC CR in your namespace, complete the following steps:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: The subscription operator cannot create an SCC **CR** automatically.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -756,23 +754,16 @@ If you cannot verify, attempt to import the cluster again and verify again.
...
- nginx-ingress-52edb
- nginx-ingress-52edb-backend
+nginx-ingress-52edb
+nginx-ingress-52edb-backend
 ----
-
-. Create an SCC CR in your namespace to assign the required permissions to the service account or accounts.
-See the following example where `kind: SecurityContextConstraints` is added:
++
+. Create an SCC CR in your namespace to assign the required permissions to the service account or accounts. See the following example where `kind: SecurityContextConstraints` is added:
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: See the following example **,** where `kind: SecurityContextConstraints` is added:

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -977,22 +962,18 @@ See link:../backup_restore/backup_restore_enable.adoc#enable-backup-and-restore[
 [#restore-limitations]
 === Velero restore limitations
 
-View the following restore limitations:
+A new hub cluster can have a different configuration than the active hub cluster if the new hub cluster where the data is restored has user-created resources. For example, this can include an existing policy that was created on the new hub cluster before the backup data is restored on the new hub cluster.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: A new hub cluster can have a different configuration than the active hub cluster if the new hub cluster, where the data is restored, has user-created resources.

**Review by dockerymick — APPROVED** [review]
Thanks for making this change! I made a couple of suggestions

**Review by birsanv — CHANGES_REQUESTED** [review]
DR changes look perfect but there is one change (change 4) not addressed here; do you plan to add that to a new PR or you just missed it ?

Add this new section
Title : Imported managed clusters show a Pending Import status

**Comment by oafischer** [comment]
@birsanv Change 4 is in line 1067, does it show up for you? Might be hard to see due to the AsciiDoc/GitHub issues in this file.

**Review by birsanv — APPROVED** [review]
got it, thanks!

lgtm

**Comment by birsanv** [comment]
@oafischer sorry, found one more change 

Replace this
Avoid backup collisions by setting the BackupSchedule.cluster.open-cluster-management.io resource status to BackupCollision. 

With 

The controller avoids backup collisions by setting the BackupSchedule.cluster.open-cluster-management.io resource status to BackupCollision. 

(The user would not be able to set the status, it is set automatically)

**Comment by oafischer** [comment]
Sorry, I'll add it in another PR! Thanks for catching that.

**Comment by oafischer** [comment]
https://github.com/stolostron/rhacm-docs/pull/4327

**Pull request merged**